subprojects/architecture-test/src/changes/archunit_store/3c3e47be-b5ed-426a-8b9f-cfa00d3b09db

```diff
@@ -25,9 +25,9 @@ Method <org.gradle.api.tasks.diagnostics.TaskReportTask.setRenderer(org.gradle.a
 Method <org.gradle.api.tasks.javadoc.Groovydoc.getAntGroovydoc()> has arguments/return type org.gradle.api.internal.tasks.AntGroovydoc that is not Gradle public API or primitive or built-in JDK classes or Groovy classes in (Groovydoc.java:198)
 Method <org.gradle.api.tasks.javadoc.Groovydoc.setAntGroovydoc(org.gradle.api.internal.tasks.AntGroovydoc)> has arguments/return type org.gradle.api.internal.tasks.AntGroovydoc that is not Gradle public API or primitive or built-in JDK classes or Groovy classes in (Groovydoc.java:207)
 Method <org.gradle.api.tasks.scala.ScalaCompile.setCompiler(org.gradle.language.base.internal.compile.Compiler)> has arguments/return type org.gradle.language.base.internal.compile.Compiler that is not Gradle public API or primitive or built-in JDK classes or Groovy classes in (ScalaCompile.java:123)
-Method <org.gradle.api.tasks.testing.Test.getTestFramework()> has arguments/return type org.gradle.api.internal.tasks.testing.TestFramework that is not Gradle public API or primitive or built-in JDK classes or Groovy classes in (Test.java:917)
-Method <org.gradle.api.tasks.testing.Test.getTestFrameworkProperty()> has arguments/return type org.gradle.api.internal.tasks.testing.TestFramework that is not Gradle public API or primitive or built-in JDK classes or Groovy classes in (Test.java:912)
-Method <org.gradle.api.tasks.testing.Test.testFramework(groovy.lang.Closure)> has arguments/return type org.gradle.api.internal.tasks.testing.TestFramework that is not Gradle public API or primitive or built-in JDK classes or Groovy classes in (Test.java:921)
+Method <org.gradle.api.tasks.testing.Test.getTestFramework()> has arguments/return type org.gradle.api.internal.tasks.testing.TestFramework that is not Gradle public API or primitive or built-in JDK classes or Groovy classes in (Test.java:915)
```

Comment:
You could revert the changes to the files in the store, they are only line number changes which are ignored by the test. This would make the PR a little bit smaller.